package.json

```diff
@@ -93,20 +93,17 @@
     "@shapeshiftoss/caip": "workspace:^",
     "@shapeshiftoss/chain-adapters": "workspace:^",
     "@shapeshiftoss/errors": "workspace:^",
-    "@shapeshiftoss/hdwallet-coinbase": "1.55.9",
-    "@shapeshiftoss/hdwallet-core": "1.55.9",
-    "@shapeshiftoss/hdwallet-keepkey": "1.55.9",
-    "@shapeshiftoss/hdwallet-keepkey-webusb": "1.55.9",
-    "@shapeshiftoss/hdwallet-keplr": "1.55.9",
-    "@shapeshiftoss/hdwallet-ledger": "1.55.9",
-    "@shapeshiftoss/hdwallet-ledger-webusb": "1.55.9",
-    "@shapeshiftoss/hdwallet-metamask": "1.55.9",
-    "@shapeshiftoss/hdwallet-native": "1.55.9",
-    "@shapeshiftoss/hdwallet-native-vault": "1.55.9",
-    "@shapeshiftoss/hdwallet-phantom": "1.55.9",
-    "@shapeshiftoss/hdwallet-shapeshift-multichain": "1.55.9",
-    "@shapeshiftoss/hdwallet-walletconnectv2": "1.55.9",
-    "@shapeshiftoss/hdwallet-xdefi": "1.55.9",
+    "@shapeshiftoss/hdwallet-core": "1.55.10-mipdalpha.1",
+    "@shapeshiftoss/hdwallet-keepkey": "1.55.10-mipdalpha.1",
+    "@shapeshiftoss/hdwallet-keepkey-webusb": "1.55.10-mipdalpha.1",
+    "@shapeshiftoss/hdwallet-keplr": "1.55.10-mipdalpha.1",
+    "@shapeshiftoss/hdwallet-ledger": "1.55.10-mipdalpha.1",
+    "@shapeshiftoss/hdwallet-ledger-webusb": "1.55.10-mipdalpha.1",
+    "@shapeshiftoss/hdwallet-metamask-multichain": "1.55.10-mipdalpha.1",
+    "@shapeshiftoss/hdwallet-native": "1.55.10-mipdalpha.1",
+    "@shapeshiftoss/hdwallet-native-vault": "1.55.10-mipdalpha.1",
+    "@shapeshiftoss/hdwallet-phantom": "1.55.10-mipdalpha.1",
+    "@shapeshiftoss/hdwallet-walletconnectv2": "1.55.10-mipdalpha.1",
```

Comment:
Alpha published on npm so we can test this, but will need to use actual when published